sdks/aperture-go/example/main.go

```diff
@@ -63,16 +62,20 @@ func grpcOptions(insecureMode, skipVerify bool) []grpc.DialOption {
 func main() {
 	ctx := context.Background()
 
-	stdr.SetVerbosity(2)
+	apertureAgentAddr := getEnvOrDefault("APERTURE_AGENT_ADDRESS", defaultAgentAddress)
```

Comment:
don't want to get anything through env. supposed to be provided explicitly by the developer. check typescript sdk

Reply:
This is example code.